lib/rubocop/cop/solidus_cops.rb

```diff
@@ -11,3 +11,4 @@
 require_relative 'solidus/spree_refund_call_perform'
 require_relative 'solidus/spree_t_decprecated'
 
+require_relative 'mixin/target_solidus_version'
```

Comment:
Tried adding the file to a `require` statement but still module is not recognised in the cop.